README.md

```diff
@@ -79,31 +79,34 @@ To see how to keep the app updated see <a href="/UPDATES.md">UPDATES.md</a>
 
 # About The Project
 
-<table>
-<tr>
-<td>
-
-Key features of **Continuum**:
-
-- Lazy mode: Automatic scrolling of posts enables you to enjoy amazing posts without moving your thumb.
-- Browsing posts
-- View comments
-- Expand and collapse comments section
-- Vote posts and comments
-- Save posts
-- Write comments
-- Edit comments and delete comments
-- Submit posts (text, link, image and video)
-- Edit posts (mark and unmark NSFW and spoiler and edit flair) and delete posts
-- See all the subscribed subreddits and followed users
-- View the messages
-- Get notifications of unread messages
-- etc...
-
-</td>
-</tr>
-</table>
-
+Unique features of **Continuum**:
+
+- Use your own `Client ID`.
+- Allow using Giphy gifs.
```

Comment:
"Ability to use your own Giphy gifs API key"

`Infinity for Reddit` does support `Giphy` out of the box.